openbb_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -9,8 +9,8 @@
 
 from openbb_terminal.core.session.current_user import get_current_user
 from openbb_terminal.cryptocurrency.defi import (
-    coindix_model,
-    coindix_view,
+    nanoly_model,
```

Comment:
It is right here where Ruff is tripping.  It wants to see these sorted alphabetically.

Reply:
There is an error with the related tests.  I think you may have to re-capture the tests for the model, view and controller.

![Screenshot 2023-12-06 at 7 43 13 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/deddf7d8-8735-46e4-bcc5-48fe82c018e9)

`pytest tests/openbb_terminal/path_to/test_file.py --record-mode once --rewrite-expected` should get us there, allegedly.

Reply:
Hi, I have tried running the command you suggested but I can't get it to run on my machine locally.
The command:
```sh
≈```

and receive this error:
```sh
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --record-mode once
  inifile: /Users/davidtran/Code/OpenBBTerminal/pytest.ini
  rootdir: /Users/davidtran/Code/OpenBBTerminal
```

I have also ran
```sh
pytest tests/openbb_terminal/cryptocurrency/test_defi_controller.py --record-mode=once --rewrite-expected
```
But received the same error.

Do you have any suggestions where to go from here?

Reply:
oooo..  looks like I missed a sub-folder (defi), sorry about that. 

tests/openbb_terminal/cryptocurrency/defi/test_defi_controller.py